<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasvurularSayfasi</class>
 <widget class="QMainWindow" name="BasvurularSayfasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>871</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>871</width>
       <height>691</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>-10</y>
        <width>461</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Mercek">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>search-button.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="arama_kutusu">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Aramak istediğiniz kelimeyi yazınız!</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="ara_Butonu">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>10</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Ara</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="TumBasvurular_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>1000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align: left; /* Metni sola hizala */</string>
       </property>
       <property name="text">
        <string>   Tüm Başvurular</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logoMini.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>361</width>
         <height>5</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(108, 108, 108);
border-color: rgb(136, 0, 0);
border-color: rgb(156, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="MentorGorTan_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align: left; /* Metni sola hizala */</string>
       </property>
       <property name="text">
        <string>   Mentor Görüşmesi Tanımlananlar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logoMini.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>361</width>
         <height>5</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(108, 108, 108);
border-color: rgb(136, 0, 0);
border-color: rgb(156, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="MentorGorTanmMa_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>1000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align: left; /* Metni sola hizala */</string>
       </property>
       <property name="text">
        <string>   Mentor Görüşmesi Tanımlanmayanlar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logoMini.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>361</width>
         <height>5</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(108, 108, 108);
border-color: rgb(136, 0, 0);
border-color: rgb(156, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="Geri_Tusu">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>600</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>Tercihler Ekranina Geri Dön!</string>
      </property>
      <property name="statusTip">
       <string extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="whatsThis">
       <string extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="accessibleName">
       <string notr="true" extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="accessibleDescription">
       <string extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../geri_tusu.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Geri_yazisi">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>650</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 255);</string>
      </property>
      <property name="text">
       <string>Geri</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Kapat_butonu_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>640</y>
        <width>45</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Uygulamadan cikar!</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>kapat.png</normaloff>kapat.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>811</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Zaman</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ad Soyad</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mail</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefon</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Posta Kodu</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Eyalet</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Su Anki Durum</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="geriButonu">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>630</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Tercih menüsüne geri döner</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>geri_tusu.png</normaloff>geri_tusu.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="liste_kisi_sayisi">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>250</y>
        <width>401</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="arama_negatif">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Geri_yazisi_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>650</y>
        <width>49</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Kapat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>730</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>©VIT | HES</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
